app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -116,4 +158,22 @@ def select_host(vm)
   def password_auth(provider)
     provider.authentication_userid_passwords.first
   end
+
+  def get_first_vm_network_id(conn, dc_id)
+    conn.system_service.data_centers_service.data_center_service(dc_id)
+        .networks_service.list.select { |net| net.usages.include? 'vm' }.first.id
```

Comment:
You can use `.detect { ... }` instead of `.select { ... }.first`.

There may be no network that satisfies the condition, so this needs to check if the result is `nil`.

Use [`OvirtSDK::NetworkUsage::VM`](http://www.rubydoc.info/gems/ovirt-engine-sdk/OvirtSDK4/NetworkUsage) instead of `'vm'`, if possible.

The results of the `list` method don't have any guaranteed order, so this may produce a result today and a different result tomorrow. I'd suggest that you first get all the networks, sort them (by name) and then get the first. That way the result will be predictable.